cfme/tests/networks/test_network_providers.py

```diff
@@ -51,7 +51,7 @@ def test_network_provider_add_with_bad_credentials(provider):
     default_credentials = provider.default_endpoint.credentials
 
     # default settings
-    flash = 'Login failed due to a bad username or password.'
+    flash = 'Login failed due to a bad username, password or unsupported API version.'
```

Comment:
**Question** Is this change in both 5.9 and 5.10? If it isn't, it will have to be picked based on the version of the appliance.

Reply:
Yes, you are right. 5.9 and earlier have old value. I pushed new changes.